src/hooks/useDydxClient.tsx

```diff
@@ -273,6 +293,10 @@ const useDydxClientContext = () => {
     [compositeClient]
   );
 
+  const getAllValidators = useCallback(async () => {
```

Comment:
[q] was this intentional? doesn't seem related to the rest of the pr

Reply:
hahaha are we just exposing this function for candidates x_x

Reply:
WOOOPS! I wanted to see the payload to help answer some questions. will remove 😂